storage.ml

```diff
@@ -86,24 +99,56 @@ module Make (BLOCK : Mirage_block.S) = struct
         in
         let* t = t_of_json json in
         Ok t
-    | Ok None -> Ok ([], Configuration.empty ())
+    | Ok None -> Ok ([], [ Configuration.empty () ])
```

Comment:
Why is that? Shouldn't we now use the empty list, and remove the "Configuration.empty" value?